packages/lodestar-cli/src/cmds/account/cmds/wallet/create.ts

```diff
@@ -18,6 +19,18 @@ interface IWalletCreateArgs {
   mnemonicOutputPath?: string;
 }
 
+function validatePassphraseFilename(passphraseFile: string): void {
+  if (path.parse(passphraseFile).ext !== ".pass") {
```

Comment:
@3xtr4t3rr3str14l If you are creating this functions just to please CodeClimate I don't think it's a good idea. In my personal opinion having to jump to another function for something this simple that is not re-used, it decreases code readability. But I would love to know others opinions.